nullability/v0.2/nullability-v0.2.graphql

```diff
@@ -0,0 +1,122 @@
+"""
+Indicates that a position is semantically non null: it is only null if there is a matching error in the `errors` array.
+In all other cases, the position is non-null.
+
+Tools doing code generation may use this information to generate the position as non-null:
```

Comment:
Potential clarification: "...as non-null _if field errors are handled out of band:_" (or something along those lines).